builds/gyp/project.gyp

```diff
@@ -80,8 +80,6 @@
         '../../src/mlm_classes.h'
       ],
       'dependencies': [
-        '../../../libzmq/builds/gyp/project.gyp:libzmq',
```

Comment:
please regenerate with libzmq and czmq checked out in the same parent directory as malamute, and with exactly those names - yes it's an annoying requirement